vmdk_plugin/drivers/photon/photon_driver.go

```diff
@@ -560,7 +560,13 @@ func (d *VolumeDriver) Create(r volume.Request) volume.Response {
 func (d *VolumeDriver) Remove(r volume.Request) volume.Response {
 	log.WithFields(log.Fields{"name": r.Name}).Info("Removing volume ")
 
-	// Docker is supposed to block 'remove' command if the volume is used. Verify.
+	if d.refCounts.GetInitSuccess() != true {
```

Comment:
Sure. IsInitialized sounds better. 👍 
But this is the Remove flow.  The volume can't be removed until refcounting is complete.
I can add a comment about this.

For the unmount, we have the comment in Unmount func.

We are hitting this error through a test while removing the volume when refcount isn't complete yet.